packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -164,7 +164,10 @@ namespace FormData {
   @doc("Test content-type: multipart/form-data for mixed scenarios")
   @post
   @route("/complex-parts")
-  op complex(@header contentType: "multipart/form-data", @body body: ComplexPartsRequest): NoContentResponse;
+  op jsonArrayAndFileArrayAndOthers(
+    @header contentType: "multipart/form-data",
+    @body body: ComplexPartsRequest,
```

Comment:
There is no `jsonArray` in the model `ComplexPartsRequest`. Do you use the wrong model as body?